app/database/models/bit_schema/user_extension.py

```diff
@@ -0,0 +1,78 @@
+from sqlalchemy import null
+from sqlalchemy.dialects.postgresql import JSONB
+from app.database.sqlalchemy_extension import db
+from app.utils.bitschema_utils import Timezone
+
+
+class UserExtensionModel(db.Model):
+    """Defines attributes for a user that are specific only to BridgeInTech.
+    Attributes:
+    user_id: A string for storing user_id.
+    is_organization_rep: A boolean indicating that user is a organization representative.
+    additional_info: A json object for storing other information of the user with the specified id.
+    timezone: A string for storing user timezone information.
+    """
+
+    # Specifying database table used for UserExtensionModel
+    __tablename__ = "users_extension"
+    __table_args__ = {"schema": "bitschema", "extend_existing": True}
+
+    id = db.Column(db.Integer, primary_key=True)
+
+    user_id = db.Column(
+        db.Integer,
+        db.ForeignKey("public.users.id", ondelete="CASCADE"),
```

Comment:
Any reason for CASCADE rather than SET NULL or any other alternative?

Reply:
Because if the user data in user table is deleted, there is no point of us keeping the data of that user (on the user_extension or personal background). All relevant data of that user needs to be deleted from the database. Otherwise we'll have an orphan data of user which serves no purpose whatsoever in the database. Does it make sense?